Umbraco-Cloud/Frequently-Asked-Questions/index.md

```diff
@@ -75,7 +75,7 @@ Generally, we recommend that you keep your DNS entry set to 'DNS Only' in your o
 
 ### What versions of .NET does Cloud support?
 
-Umbraco Cloud currently runs all projects on .NET 4.5 by default and supports up to 4.5.2, v8 projects runs on 4.7.2.
+Umbraco Cloud currently runs all projects on .NET 4.5 by default and supports up to 4.5.2, v8 projects runs on 4.7.2, and v9 projects runs on .NET 5.0..
```

Comment:
I think it makes more sense to say Umbraco 9 instead of V9 just to be clear about what version we are using :-)